client/components/domains/contact-details-form-fields/style.scss

```diff
@@ -93,7 +93,24 @@
 }
 
 .contact-details-form-fields__field.first-name,
+.contact-details-form-fields__field.location-search,
 .form__hidden-input .form__hidden-input {
 	margin-top: 0;
 }
 
+.contact-details-form-fields__field.location-search {
+	.address-1 {
+		position: relative;
+
+		&:focus {
```

Comment:
this doesn't seem to be getting picked up. We could leave it on all the time. Just looking into why it's not working or other options.